packages/grafana-ui/src/components/Collapse/CollapsableSection.story.tsx

```diff
@@ -11,13 +13,32 @@ const meta: ComponentMeta<typeof CollapsableSection> = {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['className', 'contentClassName', 'onToggle', 'labelId'],
+    },
+  },
+  args: {
+    isOpen: false,
+    loading: false,
+    label: 'Collapsable section title',
+    children: 'Collapsed content data',
+  },
+  argTypes: {
+    label: { control: 'text' },
   },
 };
 
-export const simple = () => {
+export const Basic: ComponentStory<typeof CollapsableSection> = ({ children, ...args }: Props) => {
+  const [, updateArgs] = useArgs();
+
+  const onToggle = (isOpen: boolean) => {
+    action('onToggle fired')({ isOpen });
+    updateArgs({ isOpen });
+  };
+
   return (
-    <CollapsableSection label="Collapsable section" isOpen>
-      <div>{"Here's some content"}</div>
+    <CollapsableSection {...args} onToggle={onToggle}>
```

Comment:
The collapsible section already looked a bit strange anyway being horizontal instead of vertical
<img width="756" alt="Screenshot 2022-09-09 at 12 26 43" src="https://user-images.githubusercontent.com/100691367/189339836-afaa570c-0a71-43a1-83e9-4f4af3224972.png">

I think it's because by default the storybook stories have `display: 'flex'` around them and the `CollapsableSection` is a Fragment with the two different parts inside which makes it behave badly. Can we just wrap this in a `div` to stop it from getting displayed horizontally?

Reply:
Thanks for pointing this out, I thought this collapsible section had to be horizontal. 🙌